test/vsi_test.go

```diff
@@ -0,0 +1,34 @@
+package test
+
+import (
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/terraform"
+)
+
+func TestAccIBMVSIFunction(t *testing.T) {
+	t.Parallel()
+
+	// Construct the terraform options with default retryable errors to handle the most common retryable errors in
+	// terraform testing.
+	terraformOptions := terraform.WithDefaultRetryableErrors(t, &terraform.Options{
+		// The path to where our Terraform code is located
+		TerraformDir: "../examples/vsi",
+
+		Vars: map[string]interface{}{
+			"name":                "test-vsi",
+			"vpc_name":            "test-vsi-vpc",
+			"subnet_name":         "test-vsi-subnet",
+			"location":            "us-south-2",
+			"image":               "r006-5b05b4fe-bcbc-4309-ad45-3354813227a0",
+			"profile":             "cx2-2x4",
```

Comment:
in vsi_test.go file you have hardcoded values like
"image":               "r006-5b05b4fe-bcbc-4309-ad45-3354813227a0",
"ssh_keys":            "ssh-rsa AAAAB3NzaC1yc2EAAAADAQA"
May work locally as you have these image and ssh_key locally in your machine
but test case fails once you push changes to master branch as these values will not present in repo where we run this test case

Reply:
Irrespective of account this works. so no issue with these